client/state/utils/reducer-utils.ts

```diff
@@ -3,25 +3,36 @@
  */
 import { get, mapValues, reduce } from 'lodash';
 import { combineReducers as combine } from 'redux'; // eslint-disable-line no-restricted-imports
+import type { Reducer, AnyAction, Action } from 'redux';
 
 /**
  * Internal dependencies
  */
 import { serialize, deserialize } from './serialize';
+import type { SerializableReducer } from './serialize';
 import { APPLY_STORED_STATE } from 'calypso/state/action-types';
 import { SerializationResult } from 'calypso/state/serialization-result';
 
+interface CombinedReducer extends SerializableReducer {
```

Comment:
I'm getting a bunch of those errors:

```
Declaration emit for this file requires using private name 'CombinedReducer' from module '".../client/state/utils/reducer-utils"'. An explicit type annotation may unblock declaration emit.
```

Perhaps we should export this interface as well?